utils/big_decimal.hh

```diff
@@ -9,11 +9,11 @@
 #pragma once
 
 #include "multiprecision_int.hh"
-#include <boost/multiprecision/cpp_int.hpp>
```

Comment:
I don't understand how you could remove this - cpp_int is definitely used below. Maybe you are assuming that multiprecision_int.hh includes it? But doesn't this go against the "include what you use" ideal?

Reply:
indeed. i should have replaced it with `boost/multiprecision/fwd.hpp`, but forgot to do so. should be fixed in the latest revision.